openfl-workspace/torch_cnn_mnist/README.md

```diff
@@ -0,0 +1,162 @@
+## Directory Structure
+The taskrunner workspace has the following file structure:
+```
+taskrunner
+├── requirements.txt      # defines the required software packages
+└── plan
+    ├── plan.yaml         # the Federated Learning plan declaration
+    ├── cols.yaml         # holds the list of authorized collaborators
+    ├── data.yaml         # holds the collaborator data set path
+    ├── defaults          # path to the default values for the FL plan
+├── src
+    ├── __init__.py       # treat src as a Python package
+    └── cnn_model.py      # CNN model for federated learning.
```

Comment:
nit - `# centralized CNN model, ready for use in federated learning`

Reply:
resolved in d8dda083b7c32d0680ef2d87564c3fa64b888f19